### PR TITLE
Update automl-dnn-vision-gpu with changes related to 1.53.1 pypi release

### PR DESCRIPTION
**Pull request by PhaniShekhar**

### CODE REVIEW

File: assets/training/finetune_acft_image/components/pipeline_components/classification/spec.yaml [hunk 1]
```diff
@@ -262,7 +262,7 @@ jobs:
 
   image_classification_runtime_component:
     type: command
-    component: azureml:train_image_classification_model:0.0.4
+    component: azureml:train_image_classification_model:0.0.5
```
Comment (sharma-riti) [inline assets/training/finetune_acft_image/components/pipeline_components/classification/spec.yaml:265]: don't we need to update the version of this component as well?

File: assets/training/vision/components/object_detection/spec.yaml [hunk 2]
```diff
@@ -230,7 +230,7 @@ outputs:
 
 code: ../src
 
-environment: azureml://registries/azureml/environments/automl-dnn-vision-gpu/versions/1
+environment: azureml://registries/azureml/environments/automl-dnn-vision-gpu/versions/2
```
Comment (sharma-riti) [inline assets/training/vision/components/object_detection/spec.yaml:233]: same for this... update the environment version?

Comment (PhaniShekhar) [reply assets/training/vision/components/object_detection/spec.yaml:233]: I already updated to version 2. Could you please check the latest commits.

File: assets/training/finetune_acft_image/components/pipeline_components/classification/spec.yaml (hunk 1 above)
Comment (PhaniShekhar) [reply assets/training/finetune_acft_image/components/pipeline_components/classification/spec.yaml:265]: I already updated to version 0.0.5. Could you please check the latest commits.

Comment (sharma-riti) [reply assets/training/finetune_acft_image/components/pipeline_components/classification/spec.yaml:265]: Sorry.. I meant the pipeline component version - https://github.com/Azure/azureml-assets/blob/ee3af030d0d48ef48b23f01efb0b27c4a878b51c/assets/training/finetune_acft_image/components/pipeline_components/classification/spec.yaml#L4C1-L4C15

Comment (PhaniShekhar) [reply assets/training/finetune_acft_image/components/pipeline_components/classification/spec.yaml:265]: This version was updated by Shubham to 0.0.8 along with fintune components, but this version was never released. So, using 0.0.8 for the next release.

**Review by raduk — APPROVED** [review]

**Review by sharma-riti — APPROVED** [review]

**Review by bhimar — APPROVED** [review]

**Comment by PhaniShekhar** [comment]
Version 0.0.4 for runtime components was accidentally deleted from registries.
This would fail all AutoML runs that would go through pipeline component route. Example run : https://ml.azure.com/runs/happy_dolphin_6w9y64q5f2?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/madhumaddi-aml-demo&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

Re-creating the same component with same version number 0.0.4 is not supported.
With this PR, we will re-add runtime components with version 0.0.5 using new runtime component environment. This would fix the above issue too.

File: assets/training/vision/components/image_classification/spec.yaml [hunk 3]
```diff
@@ -188,7 +188,7 @@ outputs:
 
 code: ../src
 
-environment: azureml://registries/azureml/environments/automl-dnn-vision-gpu/versions/1
+environment: azureml://registries/azureml/environments/automl-dnn-vision-gpu/versions/3
```
Comment (PhaniShekhar) [inline assets/training/vision/components/image_classification/spec.yaml:191]: Nikolay released this env with version 2, that is still pointing to 1.53.0. Hence skipping that version and using the next version 3 here.

**Review by raduk — APPROVED** [review]

**Review by sharma-riti — APPROVED** [review]

**Review by bhimar — APPROVED** [review]

**Pull request merged**